src/DynamoPackages/DynamoPackages.csproj

```diff
@@ -9,7 +9,7 @@
     <RootNamespace>DynamoPackages</RootNamespace>
     <AssemblyName>DynamoPackages</AssemblyName>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
```

Comment:
so, will net6 builds be able to load config files?

Reply:
Yes...there is a separate ConfigurationManager nuget package for that